docs/pages/setup/operations/tls-routing.mdx

```diff
@@ -13,6 +13,14 @@ description: How to upgrade exising Teleport cluster to a single-port TLS routin
   TLS routing is available starting from Teleport `8.0`.
 </Details>
 
+<Notice scope={["cloud"]} type="tip">
+
+Teleport Cloud manages the Proxy Service's networking configuration for you. You
```

Comment:
Might be worth adding whether or not TLS routing is enabled in cloud.

If it is enabled, then you can say "this is already enabled for you and you can use it today" and if it is not enabled you can say "sorry, this feature does not currently work on Teleport Cloud."